Gitcoin/DraggableViewBackground.m

```diff
@@ -1,229 +0,0 @@
-//
-//  DraggableViewBackground.m
-//  RKSwipeCards
-//
-//  Created by Richard Kim on 8/23/14.
-//  Copyright (c) 2014 Richard Kim. All rights reserved.
-//
-//
-// Forked by Kevin Owocki on 10/14/2017
-// Thanks Richard for your hard work on this.
-// Posted online open source at https://github.com/gitcoinco/ios
-
-#import "DraggableViewBackground.h"
-
-@implementation DraggableViewBackground{
-    NSInteger cardsLoadedIndex; //%%% the index of the card you have loaded into the loadedCards array last
-    NSMutableArray *loadedCards; //%%% the array of card loaded (change max_buffer_size to increase or decrease the number of cards this holds)
-    
-    UIButton* menuButton;
-    UIButton* messageButton;
-    UIButton* checkButton;
-    UIButton* xButton;
-}
-//this makes it so only two cards are loaded at a time to
-//avoid performance and memory costs
-static const int MAX_BUFFER_SIZE = 2; //%%% max number of cards loaded at any given time, must be greater than 1
-static const float CARD_HEIGHT = 386; //%%% height of the draggable card
-static const float CARD_WIDTH = 290; //%%% width of the draggable card
-
-@synthesize cardData; //%%% all the labels I'm using as example data at the moment
-@synthesize allCards;//%%% all the cards
-
-- (void)alertView:(UIAlertView *)alertView
-clickedButtonAtIndex:(NSInteger) buttonIndex{
-
-    exit(0);
-}
-
-- (id)initWithFrame:(CGRect)frame
-{
-    self = [super initWithFrame:frame];
-    if (self) {
-        [super layoutSubviews];
-        [self setupView];
-
-        NSError *error;
-        NSString *url_string = [NSString stringWithFormat: @"https://gitcoin.co/api/v0.1/bounties?&idx_status=submitted&order_by=-web3_created"];
```

Comment:
i think this should be `https://gitcoin.co/api/v0.1/bounties/?&idx_status=open&order_by=-web3_created`

i migrated all the 'submitted' statuses to be 'open'

Reply:
fixed in c7fc847